generators/client/templates/angularjs/src/main/webapp/swagger-ui/_index.html

```diff
@@ -99,7 +99,7 @@
                     window.swaggerUi.api.clientAuthorizations.add("key", apiKeyAuth);
                 <%_ } _%>
                 <%_ if (authenticationType === 'jwt' || authenticationType === 'uaa') { _%>
-                    var authToken = JSON.parse(localStorage.getItem("jhi-authenticationToken") || sessionStorage.getItem("jhi-authenticationToken"));
+                    var authToken = JSON.parse(localStorage.getItem("jhi-authenticationtoken") || sessionStorage.getItem("jhi-authenticationtoken"));
```

Comment:
@jdubois @cicoub13 :
- I think this file is correct and should not be changed -> angularjs
- you should edit the file inside angular folder